### PR TITLE
nodecontroller.go: delete redundant para.

### PR DESCRIPTION
**Pull request by zhangxiaoyu-zidif**
**What this PR does / why we need it**:

  nodecontroller.go: delete redundant para.
  para. 'node' is not used.
  golang did not warn because it has a same name para. before.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```
NONE

### CODE REVIEW

File: pkg/controller/node/nodecontroller.go [hunk 1]
```diff
@@ -505,7 +505,7 @@ func (nc *NodeController) onNodeDelete(originalObj interface{}) {
 			glog.Errorf("Received unexpected object: %v", obj)
 			return
 		}
-		node, ok = deletedState.Obj.(*v1.Node)
+		_, ok = deletedState.Obj.(*v1.Node)
```
Comment (wojtek-t) [inline pkg/controller/node/nodecontroller.go]: I don't think this is correct.

Comment (zhangxiaoyu-zidif) [inline pkg/controller/node/nodecontroller.go]: @wojtek-t The quesion is that if para. node is be used later. If used, I am wrong and I will close this PR, if not used, the para. node is redundant. Do you think so?

Comment (wojtek-t) [inline pkg/controller/node/nodecontroller.go]: Yes - the node is used later, e.g. in lines 515 and 517.

Comment (zhangxiaoyu-zidif) [inline pkg/controller/node/nodecontroller.go]: @wojtek-t No, in fact that node is defined in line 499. the original line 508's node is defined in the if of line 502. They are different. I passed the UT in my env.

Comment (gmarek) [inline pkg/controller/node/nodecontroller.go]: @wojtek-t is right - 508 is not defining new Node object, but sets the value of one defined in 499.

Comment (zhangxiaoyu-zidif) [inline pkg/controller/node/nodecontroller.go]: @wojtek-t @gmarek  Sorry, I am wrong.

**Pull request closed without merging**